index.html

```diff
@@ -792,7 +792,7 @@ <h2>Hang Tight</h2>
 	    		  clearInterval(timer); 
 	    		  v.ocean = v.ocean_target;
 	    		}
-	    		view.uniform('sealevel_mod', v.ocean);
+	    		view.uniform('sealevel_mod', (v.ocean)?(1.0):(-1.0));
```

Comment:
looks like the bug this fixes was introduced after I changed the mantleDensity to a value that allowed displacement to become negative. Nice catch.